setup-tests.js

```diff
@@ -0,0 +1,3 @@
+import { configure } from "@testing-library/react";
```

Comment:
**question**: do you need this file?

It seems like the same logic is inside `testUtils/setup.ts`.

Reply:
whooo i accidentally committed that, thanks!